twine/repository.py

```diff
@@ -132,7 +142,7 @@ def _upload(self, package):
                 allow_redirects=False,
                 headers={'Content-Type': monitor.content_type},
             )
-            bar.done()
+            bar.close()
```

Comment:
Can `tqdm.tqdm` be used as a context manager? If so it would be nice to use it like one so that `.close()` is always called, even on exception.